Engine/Setup/ConsoleSetupHandler.cs

```diff
@@ -176,6 +180,23 @@ public bool Setup(IAlgorithm algorithm, IBrokerage brokerage, AlgorithmNodePacke
         }
 
         /// <summary>
+        /// Get the available data feeds from config.json,
+        /// If none available, throw an error
+        /// </summary>
+        private static Dictionary<SecurityType, List<TickType>> GetConfiguredDataFeeds()
```

Comment:
This method exists in two places, ConsoleSetupHandler and BrokerageSetupHandler. I think it's fine until we have to replicate it once more, then maybe a SetupHandler helper class might be appropriate? This logic needs to run before IAlgorithm.Initialize() is called. I think that ISetupHandler is the best place to use this logic, but I don't see a lot of other options for places to hang it so that ISetupHandler will have access to it.